ddtrace/opentelemetry/options.go

```diff
@@ -0,0 +1,46 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package opentelemetry
+
+import (
+	"context"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+)
+
+type contextOptionsKey struct{}
+
+var startOptsKey = contextOptionsKey{}
+
+// ContextWithStartOptions returns a copy of the given context which includes the span s.
```

Comment:
```suggestion
// ContextWithStartOptions returns a copy of the given context which includes the span s. This can be used to pass a context with Datadog start options to the Start function on the OTel tracer to propagate the options.
```